src/Modules/userApi.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable */
+import foodsUI from "./displayUi.js";
+
+const url =
+  "https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/";
+const key = "4ZfW9Nw9KyQJZ8SztYPH";
+
+const getLikes = async () => {
+  const result = await fetch(`${url}${key}/likes`);
+  const data = await result.json();
+  return data;
+};
+
+const addLike = async (id) => {
+  const body = JSON.stringify({ item_id: id });
+  const headers = { "Content-type": "application/json; charset=UTF-8" };
+  /* eslint-disable */
+
```

Comment:
- Please ensure users can like a meal. Currently I'm unable to, kindly fix this 👍🏽